src/main/java/uk/gov/ons/fwmt/job_service/exceptions/types/FWMTCommonRuntimeException.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ons.fwmt.job_service.exceptions.types;
+
+import uk.gov.ons.fwmt.job_service.exceptions.ExceptionCode;
+
+public class FWMTCommonRuntimeException extends RuntimeException {
```

Comment:
why two class:
FWMTCommonRuntimeException and FWMTCommonException.
One exception is enough